api/models.py

```diff
@@ -241,7 +241,7 @@ def validate_params(cls, params: dict):
             return False, f"Invalid result value '{result}'"
 
         parent = params.get('parent')
-        if parent:
+        if parent and parent != 'null':
```

Comment:
I don't get this, why not just translate `null` as `None` when handing the query?  That way we keep the `null` special value as something for the API and the rest of the code doesn't have to change.

Reply:
Yes, translating `null` to `None` would make further validation and translation easier.
I will address this soon.